torch/distributed/launch.py

```diff
@@ -198,6 +158,27 @@ def parse_args():
     parser.add_argument('training_script_args', nargs=REMAINDER)
     return parser.parse_args()
 
+
+def pool_processes(processes, cmd):
```

Comment:
cmd is unused variable here (but you'll have to if you want to throw the `CalledProcessError` (see my comments below) you'll also have to pass a `List[cmd]` one for each process or a list of `(process, cmd)` tuple. Or alternatively you can get the cmd of the process with `process.args`